src/openms/include/OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/IsotopeDistribution.h

```diff
@@ -221,6 +221,8 @@ namespace OpenMS
     //@{
     /// operator which access a cell of the distribution and wraps it in SpectrumFragment struct
     Peak1D& operator[](const Size& index){ return distribution_[index];}
+    /// const operator which access a cell of the distribution and wraps it in SpectrumFragment struct
```

Comment:
`const operator to access a cell....` for extra bonus points (also for the non const version)